src/Raven.Client/Documents/Queries/IndexQuery.cs

```diff
@@ -166,12 +165,7 @@ public int PageSize
         }
 
         /// <summary>
-        /// Used to calculate index staleness. When set to <c>true</c> CutOff will be set to DateTime.UtcNow on server side.
-        /// </summary>
-        public bool WaitForNonStaleResultsAsOfNow { get; set; }
-
-        /// <summary>
-        /// CAUTION. Used by IDocumentSession ONLY. It will have NO effect if used with IDatabaseCommands or IAsyncDatabaseCommands.
+        /// When set to <c>true</c>> server side will wait until result are non stale or until timeout
```

Comment:
We should leave the comment from WaitForNonStaleResultsAsOfNow here?

Reply:
We don't, It's not true anymore

Reply:
The behavior of WaitForNonStaleResults now matches the behavior from WaitForNonStaleResultsAsOfNow so the description from WaitForNonStaleResultsAsOfNow should be used?

Reply:
It wasn't accurate before